src/io_threads.c

```diff
@@ -364,7 +365,7 @@ int trySendWriteToIOThreads(client *c) {
     /* Nothing to write */
     if (!clientHasPendingReplies(c)) return C_ERR;
     /* Currently, replica/master writes are not offloaded and are processed synchronously. */
```

Comment:
The comment needs an update. Write to primary is now offloaded.

To replicas, the writes are still non-blocking IO, not synchronous IO as a reader might think. Perhaps we just don't need to mention that part.

```suggestion
    /* Currently, replica writes are not offloaded to IO threads. */
```

Reply:
Thanks, changed.